test/integration/validation/chain_validator_storage_test.cpp

```diff
@@ -12,6 +12,7 @@
 #include "cryptography/crypto_provider/crypto_defaults.hpp"
 #include "cryptography/default_hash_provider.hpp"
 #include "cryptography/keypair.hpp"
+#include "framework/result_fixture.hpp"
```

Comment:
No need to include this file since `ValueOf` is available in `common/result.hpp`.